server/middlewares/controllersValidator/ReqBodyValidator.js

```diff
@@ -0,0 +1,70 @@
+import Validator from '../../helpers/Validator';
+
+class ReqBodyValidator {
+  static validateFood(req, res, next) {
+    const {
+      name,
+      description,
+      price,
+      imageurl
+    } = req.body;
+
+    if (!name || !description || !price || !imageurl) {
+      return res.status(400).json({
+        message: 'Bad request, no field should not be missing',
```

Comment:
I think it would be nice if you also send back a help message. 
e.g
helpMsg: "name: string, description: string, price: integer, imgurl: string" or something like that, some whoever is using ur endpoint would know exactly what to do